src/services/OswServiceTasks.cpp

```diff
@@ -11,23 +12,25 @@
 #include "services/OswServiceTaskWebserver.h"
 
 namespace OswServiceAllTasks {
-// OswServiceTaskExample example;
```

Comment:
Hey, this should stay - for anyone wanting to fiddle around with the example task (and learn how to use our services)!

Reply:
it's a few lines lower but it's still there.

Reply:
Whopps, missed that on my second review! :facepalm: